packages/appstore/src/components/modals/real-wallets-upgrade/real-wallets-upgrade.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Modal, DesktopWrapper, MobileDialog, MobileWrapper, Button } from '@deriv/components';
+import { useStores } from 'Stores/index';
+import './real-wallets-upgrade.scss';
+import IntroducingWallets from './components/introducing-wallets';
+import HowItWorksWallets from './components/how-it-works-wallets';
+import TradingAccountsWallets from './components/trading-accounts-wallets';
+import { ContentFlag } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+
+const RealWalletsUpgrade = () => {
+    const { traders_hub } = useStores();
+    const { is_real_wallets_upgrade_on, closeRealWalletsUpgrade, content_flag } = traders_hub;
+    const [currentStep, setCurrentStep] = React.useState(1);
+
+    const eu_user = content_flag === ContentFlag.EU_REAL;
+
+    const handleNext = () => {
```

Comment:
I think it's better to use `wizard` component from `deriv/ui` to handle the wizard.

Reply:
@hamid-deriv the deriv-ui component does not have the expected behaviour for this design and need some adjustments to be reusable here. For this sprint we want to use the current approach with `Modal` component which we already have in deriv-app for sign-up wizard.